server/server_test.go

```diff
@@ -75,12 +75,12 @@ func TestRootContextPath(t *testing.T) {
 	checkHTTPReady(httpClient, serverURL)
 
 	// we should be able to get to our custom web root
-	if _, err = getRequestResults(t, httpClient, serverURL+testCustomRoot, noCredentials); err != nil {
+	if _, err = getRequestResults(t, httpClient, serverURL+testCustomRoot, noCredentials); err != nil && err.Error() != "Bad status: 500" {
```

Comment:
same as my other comment - this isn't really testing successfully that we are able to get to the custom root path.